platform/jvm/ShellArgs.kt

```diff
@@ -0,0 +1,19 @@
+package com.vaticle.bazel.distribution.platform.jvm
```

Comment:
Shell file was moved to `common` but we want to keep these args in `platform.jvm`.